.docker/ssh_key_rsa

```diff
@@ -0,0 +1,49 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
🗒️ This is the key used to authenticate SSH connections to the ephemeral Docker container during Cucumber tests. This key is used for no other purpose, so there is no problem checking it into the Git repo.